bin/emqx

```diff
@@ -366,9 +372,9 @@ case "$1" in
                           "$(relx_start_command)"
 
         WAIT_TIME=${WAIT_FOR_ERLANG:-15}
-        while [ $WAIT_TIME -gt 0 ]; do
+        while [ "$WAIT_TIME" -gt 0 ]; do
             if ! relx_nodetool "ping" >/dev/null 2>&1; then
-                WAIT_TIME=`expr $WAIT_TIME - 1`
```

Comment:
I was wrong here. Fixing it

Reply:
`WAIT_TIME=$(("$WAIT_TIME" - 1))` -> `WAIT_TIME=$((WAIT_TIME - 1))` because it is an arithmetic variable